src/nyc_trees/apps/core/models.py

```diff
@@ -88,6 +95,10 @@ def clean(self):
     def get_absolute_url(self):
         return reverse('user_detail', args=[self.username])
 
+    def completed_training(self):
```

Comment:
Are you suggesting we delete that boolean field as well? It's not currently used, except in this helper
```
def user_has_online_training(user):
    return user.is_authenticated() and user.online_training_complete
```

We could delete the boolean field and always use the model method. Or we can pull it off into a helper. I don't know if the view level is the best place to determine this, it seems like we're going to have multiple places that check if the user has completed training.